### ISSUE TITLE
likn.xyz

### ISSUE THREAD

**Issue by ghajini** [uBlockOrigin/uAssets#2599]
Replace the bracketed [...] placeholders with your own information.

### URL(s) where the issue occurs

`http://likn.xyz/WALvz`

### Describe the issue

antiadblock+ popups

### Screenshot(s)

![screenshot 5](https://user-images.githubusercontent.com/20338483/41511055-4002439a-728d-11e8-9109-2ba7624c86c7.png)

### Versions

- Browser/version: chrome 67
- uBlock Origin version: latest

### Settings

-  uBO's default settings

**Comment by mapx-** [comment]
test these filters
```
likn.xyz##script:inject(nano-setInterval-booster.js)
likn.xyz##script:inject(abort-on-property-read.js, blockAdBlock)
likn.xyz##script:inject(abort-on-property-write.js, Fingerprint2)
sloomp.space##script:inject(popads-dummy.js)
```

**Comment by krystian3w** [comment]
When win with Google recaptcha I can get link, but pop-up's still attack.

Firefox 61.0b14, uBO 1.16.10

**Comment by lain566** [comment]
There are still popups when I click anywhere

**Comment by mapx-** [comment]
`likn.xyz `redirest me to `sloomp.space`
What about you ?

**Comment by zaphodbeebblebrox** [comment]
I get `likn.xyz`  It is fixed for me with `likn.xyz##script:inject(abort-on-property-write.js, _pop)` @lain566, does this work for you?

**Comment by okiehsch** [comment]
Yes, `likn.xyz##script:inject(abort-on-property-write.js, _pop)` is also needed on my end.

**Comment by mapx-** [comment]
added https://github.com/uBlockOrigin/uAssets/commit/638de04d6f558d2e459d80dc1db896c860f1beac

**Comment by ghajini** [comment]
still got popup after captcha verification and clicking `get link`
![screenshot 9](https://user-images.githubusercontent.com/20338483/41546026-28e7170a-733a-11e8-8903-4f731bd4dd8e.png)

**Comment by zaphodbeebblebrox** [comment]
I only got `likn.xyz` twice before it started redirecting me to `sloomp.space`. Can anyone who gets `likn.xyz` tell me if `likn.xyz##script:inject(addEventListener-defuser.js, /(click|mousedown)/)` works?

**Comment by okiehsch** [comment]
`likn.xyz##script:inject(window.open-defuser.js)` works on my end.

`/(click|mousedown)/)` should be changed to `/^(?:click|mousedown)$/`, see https://github.com/uBlockOrigin/uAssets/commit/7a62a28ddb81ef062230897ec9765f2ac01fe089#commitcomment-29225495

**Comment by KiLL4FuNK** [comment]
ADS

`http://likn.xyz/FOyi0`

**Comment by ghajini** [comment]
Popup ads back

`http://likn.xyz/WALvz`
